cadc-util/src/main/java/ca/nrc/cadc/net/ExpectationFailedException.java

```diff
@@ -65,117 +65,24 @@
 ************************************************************************
 */
 
-package ca.nrc.cadc.caom2.version;
+package ca.nrc.cadc.net;
 
-
-import ca.nrc.cadc.caom2.persistence.UtilTest;
-import ca.nrc.cadc.db.ConnectionConfig;
-import ca.nrc.cadc.db.DBConfig;
-import ca.nrc.cadc.db.DBUtil;
-import ca.nrc.cadc.util.Log4jInit;
-import javax.sql.DataSource;
-import org.apache.log4j.Level;
 import org.apache.log4j.Logger;
-import org.junit.Assert;
-import org.junit.Test;
 
 /**
  *
```

Comment:
I find it nice to have a brief description of what the exception means in the class level javadoc.

Reply:
added one liner (cut and paste from http definition of 417)